repondeur/zam_repondeur/templates/amendements.html

```diff
@@ -224,18 +249,34 @@ <h5 class="card-text text-center mt-3">
         }
         tinysort(tableLines, options)
     })
-    var filterInput = table.querySelector('input')
-    filterInput.addEventListener('keyup', function (e) {
+    var articleFilter = table.querySelector('#article-filter')
+    articleFilter.addEventListener('keyup', function (e) {
         var value = e.target.value.trim()
         ;[].slice.call(tableLines).forEach(function (line) {
             if (!value) {
-                line.classList.remove("d-none")
+                line.classList.remove('hidden-article')
                 return
             }
-            if (line.dataset.value.trim() === value) {
-                line.classList.remove("d-none")
+            if (line.dataset.article.trim() === value) {
+                line.classList.remove('hidden-article')
             } else {
-                line.classList.add("d-none")
+                line.classList.add('hidden-article')
+            }
+        })
+    })
+    var avisFilter = table.querySelector('#avis-filter')
+    avisFilter.addEventListener('change', function (e) {
+        var value = e.target.value.trim()
+        ;[].slice.call(tableLines).forEach(function (line) {
+            if (value === 'all') {
+                line.classList.remove('hidden-avis')
+                return
+            }
+            if (line.dataset.avis === value) {
```

Comment:
A good candidate for [`.toggle()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Methods)?